README.rst

```diff
@@ -23,9 +23,16 @@
         :target: https://github.com/inveniosoftware/invenio-files-rest/blob/master/LICENSE
 
 
-Files download/upload REST API similar to S3 for Invenio.
+Invenio-Files-REST is a files storage module. It allows you to store and
+retrieve files in a similar way to Amazon S3 APIs.
```

Comment:
Not sure if it makes sense to keep repeating that its similar to Amazon S3, I have removed the mentions on the introduction documentation and I kept it as a reference for the configurable backends.